NEXT_CHANGES.rst

```diff
@@ -13,3 +13,29 @@ links to relevant Issues, Discussions, and PR's on github with the following for
 `PR #XX <https://github.com/ixdat/ixdat/pulls/XX>`_
 
 
+ixdat next
+========================
+
+changed
+-------
+
+parsers
```

Comment:
"readers"

Reply:
That is probably prevalent. I will grep try and grep the diff of the entire PR to look for it.